ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/ApiPackageManagementService.scala

```diff
@@ -74,30 +77,45 @@ class ApiPackageManagementService(
         FutureConverters
           .toScala(res._1)
           .flatMap {
-            case UploadPackagesResult.Ok =>
+            //TODO BH: first step to get this to compile -- need to implement properly polling and retrieval of response event
```

Comment:
TODO: Implement polling based on package upload entries and not the list of packages stored.

Reply:
Who's `BH`? :thinking:

Reply:
yeah this one is still pending -- will tackle in the a.m.

Reply:
implemented now